### PR TITLE
Fix #3527 

### LINKED ISSUE #3527: Update baseline script to support python 3
The baseline script https://github.com/zaproxy/zaproxy/blob/develop/build/docker/zap-baseline.py currently only supports python 2.
It would be good for this to be python 3 compatible as well.
Hopefully that wont be too hard for a pythonista?

### PR DESCRIPTION
**Pull request by AnamikaD**
@psiinon I changed the code such that it is compatible with python 2 and 3. Though, I get this error when I run zap-baseline.py. Not sure whether it is due to code change or improper testing.

```
time="2017-06-03T22:23:41+05:30" level=fatal msg="Post http://%2Fvar%2Frun%2Fdocker.sock/v1.18/images/create?fromImage=owasp%2Fzap2docker-weekly%3Alatest: dial unix /var/run/docker.sock: connect: permission denied. Are you trying to connect to a TLS-enabled daemon without TLS?" 
Traceback (most recent call last):
  File "zap-baseline.py", line 632, in <module>
    main(sys.argv[1:])
  File "zap-baseline.py", line 348, in main
    ls_output = subprocess.check_output(['docker', 'pull', 'owasp/zap2docker-weekly'])
  File "/usr/lib/python3.5/subprocess.py", line 629, in check_output
    **kwargs).stdout
  File "/usr/lib/python3.5/subprocess.py", line 711, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['docker', 'pull', 'owasp/zap2docker-weekly']' returned non-zero exit status 1

### CODE REVIEW

File: build/docker/zap-baseline.py [hunk 1]
```diff
@@ -53,10 +53,12 @@
...
 import time
 import traceback
-import urllib2
+from six.moves.urllib.parse import urlparse, urlencode
+from six.moves.urllib.request import urlopen
+from six.moves.urllib.error import HTTPError
 from datetime import datetime
 from random import randint
-from zapv2 import ZAPv2
+#from zapv2 import ZAPv2
```
Comment (thc202) [inline build/docker/zap-baseline.py]: Was this changed on purpose? This will lead to errors, e.g.:
```
Unexpected error: <type 'exceptions.NameError'>
Traceback (most recent call last):
  File "zap-baseline.py", line 396, in main
    zap = ZAPv2(proxies={'http': 'http://' + zap_ip + ':' + str(port), 'https': 'http://' + zap_ip + ':' + str(port)})
NameError: global name 'ZAPv2' is not defined
```

File: build/docker/zap-baseline.py [hunk 2]
```diff
@@ -459,7 +461,7 @@ def main(argv):
             logging.warning('No URLs found - is the target URL accessible? Local services may not be accessible from the Docker container')
         else:
             if detailed_output:
-                print ('Total of ' + str(num_urls) + ' URLs')
+                print(('Total of ' + str(num_urls) + ' URLs'))
```
Comment (thc202) [inline build/docker/zap-baseline.py]: Are these changes really needed?

**Comment by thc202** [comment]
That error does not seem to be caused by the changes, which OS are you using?

**Comment by thc202** [comment]
Linking to the issue, #3527.

File: build/docker/zap-baseline.py (hunk 1 above)
Comment (AnamikaD) [inline build/docker/zap-baseline.py]: @thc202 thank you so much for the review. It was my mistake. I forgot to uncomment that line.

File: build/docker/zap-baseline.py (hunk 2 above)
Comment (AnamikaD) [inline build/docker/zap-baseline.py]: Hi i will tell that without that changes. Thank you

**Comment by AnamikaD** [comment]
@thc202 I have installed python-owasp-zap-v2.4 using pip but still it says 

```
Traceback (most recent call last):
  File "zap-baseline.py", line 61, in <module>
    from zapv2 import ZAPv2
ImportError: No module named zapv2

**Comment by thc202** [comment]
Was `python-owasp-zap-v2.4` installed to the Python version being used (e.g. pip vs pip3)?

**Comment by AnamikaD** [comment]
@thc202  using pip3. I was getting weird error when used pip

**Comment by thc202** [comment]
OK. Which version are you using to run the script? Python 3 as well?

**Comment by AnamikaD** [comment]
Yes python and python3

**Comment by thc202** [comment]
Follow up pull request at #3617.
